lib/Room.ts

```diff
@@ -40,6 +40,10 @@ export const closeChat = async (modify: IModify, read: IRead, rid: string, persi
     const room: ILivechatRoom = (await read.getRoomReader().getById(rid)) as ILivechatRoom;
     if (!room) { throw new Error(Logs.INVALID_ROOM_ID); }
 
+    if (!room.isOpen) {
+        return;
+    }
```

Comment:
Can we through error here? So that we will get to know that some of our use case/flow is trying to close a room that is already closed. So that we can update logic to avoid it in first place

Reply:
Added new Error.